manim/utils/hashing.py

```diff
@@ -111,6 +111,10 @@ def _iter_check_list(lst):
             # We have to make a copy, as we don't want to touch to the original list
             # A deepcopy isn't necessary as it is already recursive.
             lst_copy = copy.copy(lst)
+            if isinstance(lst, tuple):
+                # NOTE : Sometimes a tuple can pass through this function. As a tuple is immutable, we convert it to a list to be able to modify it.
+                # It's ok as it's a copy.
```

Comment:
Suggestion: break a line that is too long and a minor reformulation:

```suggestion
                # NOTE: Sometimes a tuple can pass through this function. As a tuple
                # is immutable, we convert it to a list to be able to modify it.
                # It's ok as it is a copy.
```